code/modules/antagonists/heretic/items/hunter_rifle.dm

```diff
@@ -134,6 +134,46 @@
 	damage = 30
 	stamina = 30
 	projectile_phasing =  PASSTABLE | PASSGLASS | PASSGRILLE | PASSCLOSEDTURF | PASSMACHINE | PASSSTRUCTURE | PASSDOORS
+	///The mob that is currently inside the bullet
+	var/mob/stored_mob
+
+/obj/projectile/bullet/strilka310/lionhunter/fire(angle, atom/direct_target)
+	. = ..()
+	if(!isliving(firer) || !isliving(original))
+		return
+	var/mob/living/living_firer = firer
+	if(IS_HERETIC(living_firer))
+		living_firer.swap_hand() //Forcibly unscope
+		living_firer.swap_hand()
```

Comment:
I feel like we as well make it so the scope component unscopes if the mob is moved on a location that isn't a turf at this point.

Reply:
That sounds reasonable